R/hList.R

```diff
@@ -0,0 +1,79 @@
+################################################################################
+#' Extract checksum value
+#'
+#' The function hList extract checksum value from a list of file and return the value its respective filename as an R
+#' object. The function is for now using the algorithm 'xxhash64'.Checksum is compiled using the digest function.
+#'
+#' @param flist A vector listing object from which we want to extract chekcsum value.
+#' @param dir A character string representing path to file.
+#' @param quick A logical argument. If TRUE, checksum is compiled using the combination of the filename and its size.
+#'        If FALSE, cheksum is compiled using the object. Default is FALSE.
+#' @param csalgorithm A character string representing the algorithms used in the digest function. Default is "xxhash64".
+#'
+#' @return The hList function returns a dataframe where filename, checksum value from the object (checksumFile), checksum
+#'         value from the combination of filename and file size and the algorithm used to compute checksum values are stored.
+#'
+#' @importFrom digest digest
+#'
+#' @export
```

Comment:
should this be exported? this seems like an internal function -- not to be used by the user.